docs/api/wanchain.md

```diff
@@ -1,5 +1,255 @@
+
+In dynamic deploy, we do not care who performs the deploy. In this case, we do not set `deploy.takerId`. Now any account (wallet) will be able to perform such deploy and will automatically become its Taker.
+
+### DeployGateway(provider, deployGatewayId)
+
+A `class` which represents a smart contract on the Wanchain blockchain.
+
+**Arguments**
+
+| Argument | Description
+|-|-
+| deployGatewayId | [required] A `string` representing an address of the [0xcert deploy gateway smart contract](#public-addresses) on the Wanchain blockchain.
+| provider | [required] An instance of an provider.
+
+**Usage**
+
+```ts
+import { HttpProvider } from '@0xcert/wanchain-http-provider';
+import { DeployGateway } from '@0xcert/wanchain-deploy-gateway';
+
+// arbitrary data
+const provider = new HttpProvider();
+const deployGatewayId = '0x073d230a53bffc8295d9a5247296213298e3fbcf';
+
+// create deploy gateway instance
+const deployGateway = new DeployGateway(provider, deployGatewayId);
+```
+
+### cancel(deploy)
+
+An `asynchronous` class instance `function` which marks the provided `deploy` as canceled. This prevents the `deploy` to be performed.
+
+**Arguments:**
+
+| Argument | Description
+|-|-
+| deploy.assetLedgerData.capabilities | [required] An array of `AssetLedgerCapability` representing the capabilities of the `AssetLedger` that will be deployed.
+| deploy.assetLedgerData.name | [required] A `string` representing the name of the `AssetLedger` that will be deployed.
+| deploy.assetLedgerData.owner | [required] A `string` representing the address of owner of the `AssetLedger` that will be deployed.
+| deploy.assetLedgerData.schemaId | [required] A `string` representing the schemaId of the `AssetLedger` that will be deployed.
+| deploy.assetLedgerData.symbol | [required] A `string` representing the symbol of the `AssetLedger` that will be deployed.
+| deploy.assetLedgerData.uriBase | [required] A `string` representing the uriBase of the `AssetLedger` that will be deployed.
+| deploy.expiration | [required] An `integer` number representing the timestamp in milliseconds at which the deploy expires and can not be performed any more.
+| deploy.makerId | [required] A `string` representing the Wanchain account address which makes the deploy. It defaults to the `accountId` of a provider.
+| deploy.seed | [required] An `integer` number representing the unique deploy number.
+| deploy.takerId | A `string` representing the Wanchain account address which will be able to perform the deploy on the blockchain. This account also pays for the gas cost.
+| deploy.tokenTransferData.ledgerId | A `string` representing the value ledger address of which value will be tranfered.
+| deploy.tokenTransferData.receiverId | A `string` representing the Wanchain account address which will receive the value.
+| deploy.tokenTransferData.value | A `string` representing the value amount that will be transfered.
```

Comment:
| deploy.tokenTransferData.value | A `string` representing the value amount that will be transferred.